test/embedded/internalize-no-stdlib.swift

```diff
@@ -25,13 +25,11 @@ public func main() {
   start(p: Concrete())
 }
 
-// CHECK-ELF: @"\01l_entry_point" =
 // CHECK-ELF: @__swift_reflection_version =
 // CHECK-ELF: @_swift1_autolink_entries =
-// CHECK-ELF: @llvm.compiler.used = appending global [3 x ptr] [ptr @"\01l_entry_point", ptr @__swift_reflection_version, ptr @_swift1_autolink_entries], section "llvm.metadata"
+// CHECK-ELF: @llvm.compiler.used = appending global [2 x ptr] [ptr @__swift_reflection_version, ptr @_swift1_autolink_entries], section "llvm.metadata"
```

Comment:
Just wondering why is this `__swift_reflection_version` pointer here?

Reply:
We should remove it too, https://github.com/apple/swift/pull/70619.